testsuite/integration/iiop/src/test/config/arq/arquillian.xml

```diff
@@ -26,14 +26,14 @@
         <container qualifier="iiop-server" default="false">
             <configuration>
                 <property name="jbossHome">${basedir}/target/jbossas-iiop-server</property>
-                <property name="javaVmArguments">${server.jvm.args}</property>
+                <property name="javaVmArguments">${server.jvm2.args}  -Djboss.socket.binding.port-offset=100</property>
```

Comment:
it is not rename as such, there are two sets of properties for two servers that are started at same time.
now that xslt is not used to modify the binding props we need them separtly